### PR TITLE
📖 Re-add support page required by amp.dev

### LINKED ISSUE #2443: The standards/schema seem to require an image for blog posts?
It seems like the Schema for the AMP pages require an image for a blog post, which I don't really understand the motivation of. Why would all blog posts need to have an image? Google's Webmaster tool flags all my AMP pages without images as not valid.

Use case: I have a simple blog post about a problem I ran into and solved building a web application in Django. There by default no image for this kind of post.

### PR DESCRIPTION
**Pull request by sebastianbenz**
Fixes https://github.com/ampproject/docs/issues/2443

### CODE REVIEW

**Comment by sebastianbenz** [comment]
@cramforce can you please merge

File: SUPPORT.md [hunk 1]
```diff
@@ -1 +1,24 @@
-See our [README.md](README.md) for ways to get help with AMP.
+# Getting Support	See our [README.md](README.md) for ways to get help with AMP.
```
Comment (cramforce) [inline SUPPORT.md]: Just noticed that you probably want to delete the second part here.

Comment (cramforce) [reply SUPPORT.md]: And could you add a comment about this dependency between the website and this repro?

**Comment by sebastianbenz** [comment]
I had to create a new PR for the changes as I couldn't modify this protected branch  https://github.com/ampproject/amphtml/pull/23253

**Pull request closed without merging**